source/cli.civet

```diff
@@ -51,16 +51,36 @@ if process.argv.includes "--help"
 encoding .= "utf8" as BufferEncoding
 
 fs from "fs/promises"
+type { Stats } from fs
 path from "path"
+type { CompileOptions } from "@danielx/civet"
```

Comment:
We should use the relative path here so we get any updated types without having to update the dev dependency.

Reply:
Sure, I can point to the `.d.ts` for now. I think ideally we move the type definitions into an actual `.civet` file, and the `.d.ts` is generated (via unplugin say), but that can come later.

Reply:
Actually, I don't see how to import a type from a `.d.ts` file, because `declare module { ... }` is ambient, not importable/module compatible. ☹️ 

If you know a way, I can implement it (or feel free to). Otherwise, I think this should wait until the `.d.ts` has a build step.

Reply:
Just add a TODO comment in the meantime then.